components/ImageSlider/ImageSlider.tsx

```diff
@@ -0,0 +1,297 @@
+
+      const rightGutter = 10;
+      const currentOffset = pagerBox.left - windowBox.left;
+
+      if (targetItemBox.right > windowBox.right) {
+        setPagerOffset(currentOffset + (windowBox.right - targetItemBox.right) - rightGutter);
+      } else if (targetItemBox.left < windowBox.left) {
+        setPagerOffset(currentOffset + (windowBox.left - targetItemBox.left));
+      }
+    }, [activeSlide],
+  );
+
+  useEffect(() => {
+    adjustPagerPosition();
+  }, [activeSlide, adjustPagerPosition]);
+
+  const sliderSettings = useMemo(() => ({
+    arrows: false,
+    accessibility: true,
+    swipeToSlide: true,
+    lazyLoad: 'ondemand' as const,
+    dots: true,
+    dotsClass: 'relative @container',
+    customPaging: (i: number) => (
+      <ThumbnailButton
+        slide={slides[i]}
+        isActive={activeSlide === i}
+        onFocus={() => adjustPagerPosition(i)}
+        ariaLabel={`Go to slide ${i + 1} ${slides[i]?.alt || ''}`}
+      />
+    ),
+    afterChange: (i: number) => {
+      setActiveSlide(i);
+    },
+    // The bottom half of the Slider which includes the counter, expand button, caption and thumbnail nav.
+    appendDots: (dots: React.ReactNode) => (
+      <div>
+        <FlexBox justifyContent="center" className={styles.buttonWrapper}>
+          <NextPrevButton
+            direction="prev"
+            isLightText={isLightText}
+            onClick={clickPrev}
+            className={styles.nextButton}
+          />
+          <NextPrevButton
+            direction="next"
+            isLightText={isLightText}
+            onClick={clickNext}
+            className={styles.prevButton}
+          />
+        </FlexBox>
+        <FlexBox justifyContent="center" className={styles.counterExpandWrapper}>
+          <Text as="span" aria-hidden="true" leading="none" align="center">
+            {`${activeSlide + 1}/${slides?.length}`}
+          </Text>
+          <SrOnlyText>{`Slide ${activeSlide + 1} of ${slides?.length}`}</SrOnlyText>
+          {showExpandLink && (
+            <button
+              type="button"
+              onClick={openModal}
+              aria-haspopup="dialog"
+              className={styles.expandButton(isLightText)}
+              aria-label="Expand gallery in full screen modal"
+            >
+              Expand
+              <HeroIcon icon="expand" className={styles.expandIcon} />
+            </button>
+          )}
+        </FlexBox>
+        <RichText
+          textColor={isLightText ? 'white' : 'black-70'}
+          linkColor={isLightText ? 'digital-red-xlight' : 'unset'}
+          wysiwyg={slides[activeSlide]?.caption}
+          className={styles.caption}
+        />
+        <button
+          type="button"
+          onClick={focusLastThumb}
+          className={styles.skipButton}
+        >
+          {`Below is a navigation for ${slides?.length} total slides. Skip to the last item.`}
+        </button>
```

Comment:
Add a skip button to skip to the last thumbnail button right before the thumbnail nav

Reply:
Can you tell me more about the choice to skip to the last item vs skip past the thumbnails and on to the next content? The skip buttons we've introduced thus far skips over nav. ie: `skip to main content` or `skip to results`.

Reply:
@sherakama I did this because I prefer to keep the code scope to this component as much as possible. Generally I don't love the idea to do a query to find the next focusable item on the page since the thumbnail nav is the very last thing in the image slider. I think that having it skip to the last item in the nav is a ok compromise, since many pagination has a "last" page link too so this is similar 😄